classes/observer.php

```diff
@@ -172,11 +172,12 @@ public static function email_mention_hsu(\mod_hsuforum\event\assessable_uploaded
                         $discussion_id = $other->discussionid;
                         $post_id = $event->objectid;
                         $course_id = $event->courseid;
+                        $reply_id = $event->objectid;
```

Comment:
Is this correct? See it is also being used for post_id

Reply:
shot changed this